RHDEVDOCS-3623: Update Pipeline section in add flow and pipeline docs (#42868)
openshift/openshift-docs: modules/odc-importing-codebase-from-git-to-create-application.adoc

The *Serverless Deployment* option is displayed in the *Import from git* form only if the {ServerlessOperatorName} is installed in your cluster. For further details, refer to the {ServerlessProductName} documentation.
====

> . In the *Pipelines* section, select *Add Pipeline*, and then click *Show Pipeline Visualization* to see the pipeline for the application. You can also see a default pipeline selected for the application and you can choose the pipeline you want. Additionally, administrators can opt out of Tekton Hub integration and the developers will be able to see cluster tasks and tasks which are in the current namespace. You can check out xref:../../cicd/pipelines/understanding-openshift-pipelines.adoc[Understanding OpenShift Pipelines] to learn more about it.

Reviewer: ```suggestion
. In the *Pipelines* section, select *Add Pipeline*, and then click *Show Pipeline Visualization* to see the pipeline for the application. You can see a default pipeline selected and choose the pipeline you want from the list of available pipelines for the application. Additionally, if an administrator has opted out of Tekton Hub integration, as a developer, you can see cluster tasks and tasks which are in the current namespace.
```

Suggestion: 

- Maintain consistency in content writing. Follow the same style throughout.
- No internal links in modules. Please refer to the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references). If you have to mention it in the modules, add the link under the Additional resources section.
